vote/models.py

```diff
@@ -1,56 +1,21 @@
 from django.db import models
-from django.contrib.contenttypes.models import ContentType
-from django.contrib.contenttypes.fields import GenericForeignKey
+from vote.base_models import AbstractVote
 from vote.managers import VotableManager
+try:
+    from swapper import swappable_setting
+except ImportError:
+    swappable_setting = None
 
-UP = 0
-DOWN = 1
```

Comment:
this is also a breaking change that users' code could be broken after upgrading if they use this variable.